sdks/python/apache_beam/typehints/decorators_test_py3.py

```diff
@@ -154,5 +156,23 @@ def fn(a, b=None, *args, foo, **kwargs):
       decorators.getcallargs_forhints(fn, 5)
 
 
+class DecoratorsTest(unittest.TestCase):
+  def test_no_annotations(self):
```

Comment:
Better to split into three test methods.

Reply:
It is one test. I'll rewrite it to make it clear. First it shows that the pipeline raises a TypeCheckError with annotations and then it shows no exceptions raised without them.